### ISSUE TITLE
rms_vis returns zero instead of a width in float32's last binade

### ISSUE THREAD

**Issue by chrisfinlay** [epfl-radio-astro/tabascal#235]
`rms_vis` scales by the largest kept sample before squaring, precisely so that a large `|V|` does not overflow on its way to a mean square — the comment at [`fft_gp.py`](../blob/main/tabascal/fft_gp.py) says as much, citing 1e20 squaring to inf in float32. That works, and works well past where it needs to. It fails in the last binade.

Measured, constant real visibility, complex64, single precision:

```
std(1e+33) = 1.0000e+33   OK
std(1e+36) = 1.0000e+36   OK
std(1e+37) = 1.0000e+37   OK
std(1e+38) = 0.0000e+00   WRONG
```

`jnp.abs` on complex64 is not the culprit — it is exact at 1e38 — so the loss is further down, and the boundary sits between 1e37 and 1e38 against a float32 max of 3.4e38.

**A visibility of 1e38 Jy is not a physical measurement**, so on its own this is not worth much. What makes it worth recording is the failure *mode*: it returns **zero**, silently. A prior width of zero pins every mode, which is a catastrophic fit rather than a loud one, and this function's entire job is to be the thing that raises rather than guesses — it already refuses an all-masked input and a non-finite one by name.

It also composes with #232. The path by which an absurd magnitude reaches this function is a corrupt or flagged cell in the DATA column, which is exactly what #232 is about; a cell holding a large finite garbage value gets a zero width where a NaN in the same cell gets a clean error.

## Suggested shape

Cheapest correct fix is to make the out-of-range case loud rather than silent: check the result is finite and non-zero when the input was not identically zero, and raise naming the key, as the surrounding code already does for its other refusals. Getting the last binade arithmetically right is not worth the complexity.

While in there, note that the per-baseline median fallback — the stand-in for a baseline that kept nothing — is computed after the finiteness check, so it can propagate a value the check would have caught.

## How this was found

A retrospective Codex pass over #228, reproduced by hand. The pass also confirmed the `rfi.gp_cov.std` convention and the doubling of the shipped values are correct; this and #232 were the only defects in that commit.
